build.sbt

```diff
@@ -5,10 +5,12 @@ name := "Agora"
 
 organization := "org.broadinstitute"
 
-scalaVersion := "2.11.12"
-dependencyOverrides += "org.scala-lang" % "scala-compiler" % "2.11.12"
+scalaVersion := "2.12.4"
```

Comment:
(optional) `2.12.5` was released a few hours ago, in case you want to be super cutting-edge